lib/modulr/api/payments_service.rb

```diff
@@ -5,10 +5,18 @@ module API
     class PaymentsService < Service
       def find(id:)
         response = client.get("/payments", { id: id })
-        payment_attributes = response.body[:content]&.first
-        raise NotFoundError, "Payment #{id} not found" unless payment_attributes
+        @payment_attributes = response.body[:content]&.first
+        raise NotFoundError, "Payment #{id} not found" unless @payment_attributes
 
-        Resources::Payments::Payment.new(response.env[:raw_body], payment_attributes)
+        type =  if outgoing && !internal
```

Comment:
I'm not sure about it 🤔 What happen with incoming payments?

Reply:
Yes, but inside details, we should maintain at least same structure, isn't it? 🤔